app/gui/src/model/execution_context.rs

```diff
@@ -216,6 +217,78 @@ pub struct LocalCall {
 
 
 
+// ==============================
+// === QualifiedMethodPointer ===
+// ==============================
+
+/// A method pointer containing the qualified module and type names.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct QualifiedMethodPointer {
+    /// A module name containing the method.
+    pub module:          module::QualifiedName,
+    /// A type on which the method is defined.
+    pub defined_on_type: tp::QualifiedName,
+    /// A method name.
+    pub name:            String,
```

Comment:
There are stronger types like `Identifier` or `NotReferentName` available. Consider using appropriate one, if it is not too much hassle.

Unless there are more syntactically complex names that can appear here?